db/seeds/prod.seeds.json

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
Hopefully the seeder throws an exception for a Rails production env - should it be impossible to use seeds on a production system?  It might be safer to disable seeds on prod.